pkg/types/types.go

```diff
@@ -129,6 +132,7 @@ type RunSpec struct {
 	DeviceMappings []DeviceMapping `json:"DeviceMappings"` //required for all compilers
 	// DefaultInstanceMemory is in MB
 	DefaultInstanceMemory int                `json:"DefaultInstanceMemory"` //required for all compilers
+	MinInstanceDiskMB     int64              `json:"MinInstanceDiskMB"`
```

Comment:
I have forgotten how we set the capstan OpenStack: This is used to find the appropriate flavour?